src/content/thought/the-state-of-ai-discourse.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "The state of AI discourse"
+description: "It’s possible to believe AI is one of the most transformative tools of our time and that most of what’s being sold as AI is smoke and mirrors. Both can be true."
+pubDate: "04 Jun 2025 13:00:00"
+---
+
+For a while now, I've struggled to hold uncomfortable feelings about AI in tension with the reality of AI within our industry and world today. Thanks to a post by [Steve](https://steveklabnik.com/writing/i-am-disappointed-in-the-ai-discourse/), I feel compelled to try and articulate my current stance on the subject.
+
+Do not fear, this isn't a [hostile take-down of why you're not smart enough](https://fly.io/blog/youre-all-nuts/) to see the value in AI. Instead, I'll try to provide balanced reasoning on how we can improve the state of conversation around it.
+
+tl;dr:
+
+> It’s possible to believe AI is one of the most transformative tools of our time and that most of what’s being sold as AI is smoke and mirrors. Both can be true.
+
+A familiar narrative has formed around AI in our industry, one that continues to divide creators in our space. Like Steve, I find myself disappointed, not in AI itself, but in how we talk about it. The snake oil is easy to spot if you’re close to the tech, but harder for the average person wading through a timeline of grifters on LinkedIn.
+
+AI is now a well established resident in the tower block of technology, and will leave a lasting legacy on software engineering for years to come. As always, there is a disconnect between the marketing hype and the technical reality that we as technologists are faced with.
+
+
+## Signal-to-noise ratio
+
+One of the things that detracts from good discourse on AI is the involuntary reaction a lot of companies have had to feeling the need to inject AI into every part of their product. Functionality that existed before the rise of AI re-emerges as new functionality _now with 100% more AI_. A cunning move played by marketeers to demonstrate speed to market on meaningful functionality, but sadly a move which is fooling no one.
+
+Dunking on people or companies promoting AI may occasionally have merit, when there is clear exploitation or behaviour from bad actors, but it's important that we don't lose track of potentially relevant changes going on behind the scenes.
+
+There are plenty of skilled teams actively trying to build meaningful products that solve real world problems, but it's hard to focus solely on this when aspects of our industry are also being threatened. There is the hugely important debate of creativity in our industry, and how there is pride and enjoyment out of the act of creating vs shipping lines of code (which in case you're new here, is never a good metric of productivity or value).
```

Comment:
I think a good para to follow this one with would be one that acknowledges that it is the small handful of big tech companies that are doing a lot of the harms and they should be the ones on the receiving end of the heat.